cli.py

```diff
@@ -59,6 +60,8 @@ def main(log_level):
 
 
 @main.command()
+@click.option('-b', 'base_setup_path', default=None, type=str,
+              help="Accept absolute or relative path for client script")
```

Comment:
Since you used this for the `--simulation-info`:
> Simulation File info (accept absolute and relative path)
it would be good to also have something like
> Client script (accepts absolute and relative path)

Also, I think it would be nice to add a description of the `base_setup_path` and `simulation_info` usage in the README, since the difference is not immediately clear from the cli arguments and help strings.

Reply:
Sure, will do once finalised.

Reply:
Added: https://github.com/gridsingularity/d3a-api-client/pull/94/commits/63e7237b5f1b757983047df2f81154ab801aa569

Reply:
Great addition to the docs! What about slightly changing the `help` value here as well? Something like:
```suggestion
              help="Path of the folder containing client scripts (accepts absolute and relative path)")
```